### PR TITLE
Fix some deprecated behavior & stale dependencies

### PR DESCRIPTION
**Pull request by SamClayton**
With these small changes, the app will now run under React 16.0.0, opening a path to applying other security & bugfix dependency releases. I didn't include the updated package.json in this PR, since it could potentially break things that I'm not noticing on my local, though the React dev blog claimed the upgrade should require minimal code changes for existing React 15.6 apps.

### CODE REVIEW

**Comment by felixlinker** [comment]
Thanks for the work! Two things: You claim that the `package.json` was not updated, however, I see it as changed?

Second, can you summarize the changes to `VerticalBarChart.jsx`?

File: src/app/components/VerticalBarChart.jsx [hunk 1]
```diff
@@ -1,80 +1,81 @@
...
-    if (this.props.data[i].tooltip) {
-      return this.context.termtip(this.props.data[i].tooltip, e);
-    } else {
-      return null;
-    }
-  }
-}
+import TranslatedComponent from './TranslatedComponent';
+import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (SamClayton) [inline src/app/components/VerticalBarChart.jsx:3]: Use 'new' prop-types package instead of deprecated import from react.

**Comment by SamClayton** [comment]
> Thanks for the work! Two things: You claim that the package.json was not updated, however, I see it as changed?

I forgot that Github continues adding commits on a given branch to a pull request, even after it's been submitted. At the time I wrote that comment, the package.json hadn't been updated, as I was being particularly conservative about changes. I'll keep that in mind for future feature branches.
Because of that, the scope has changed. Now, this also includes the dependency updates as well as changes to build/deploy that allow the app to run on Heroku for staging-level testing/debugging and sharing in-progress versions of the app. `npm start` now serves the static `build` directory from a very simple Express server (similar to your prod deploys, just Express instead of nginx) to avoid hot-reloading in a non-local environment. The webpack hotloading build meant for local dev can still be triggered with `npm run start-dev`. Overall, it's still a pretty modest changeset.

>Second, can you summarize the changes to VerticalBarChart.jsx?

It looks like the `VerticalBarChart.jsx` comparison suffered from a difference in line-ending characters between my IDE and the repo in commit 0d8cdf9. I added a code review comment on the 2 lines changed, which should appear in this PR now. In short, PropTypes is imported from a different package in order to clear up a breaking deprecation. No other changes were made.

**Comment by felixlinker** [comment]
Okay, so coming back here: The update to how prop types are imported - would we need to cascade this to all files?

**Comment by felixlinker** [comment]
Closed because inactive. Happy to re-open anytime!
